submissions/ik-web/frogger/app.js

```diff
@@ -0,0 +1,198 @@
+const areaWidth = 505;
+const indentBorderLeft = 3;
+const indentBorderTop = 3;
+const indentBorderRight = 404;
+const indentBorderBottom = 370;
+const levelForWin = 12;
+// ---------------------------------------------
+const playerSprite = 'images/char-horn-girl.png';
+const playerStartLevel = 1;
+const playerStartPosX = 205;
+const playerStartPosY = 380;
+const playerStepX = 101;
+const playerStepY = 80;
+// ---------------------------------------------
+const borderCollision = 50;
+// ---------------------------------------------
+const enemySprite = 'images/enemy-bug.png';
+const enemySpeedMax = 500;
+const enemySpeedMin = 150;
+//----------------------------------------------
+const enemyPosX = -100;
+const enemyPosY = function() {
+    const initYPositions = [60, 140, 220];
+    let randY = Math.floor( Math.random() * 3);
+
+    return initYPositions[randY];
+};
+//----------------------------------------------
+const levelCounter = document.createElement('div');
+//----------------------------------------------
+
+// Generate enemies' speed
+function enemySpeed() {
+    return Math.floor( Math.random() * (enemySpeedMax - enemySpeedMin) + enemySpeedMin);
+}
+
+// Add a new enemy after level up
+function addEnemy() {
+    let enemy = new Enemy(enemyPosX, enemyPosY(), enemySprite, enemySpeed(), player);
+    allEnemies.push(enemy);
+}
+
+// Clear enemies if the player won or lose
+function clearEnemy() {
+    allEnemies = [];
+    addEnemy();
+}
+
+//Create level counter
+function addLevelCounter() {
+    levelCounter.style.cssText = `
+        position: fixed; 
+        top: 10px; 
+        left: 50%;
+        font-family: sans-serif;
+        font-size: 30px; 
+        transform: translateX(-50%);
+    `;
+
+    levelCounter.innerHTML = `Level: ${playerStartLevel}`;
+    document.body.prepend(levelCounter);
+}
+
+//----------------------------------------------
+// Constructor for create character
+const Character = function(x, y, sprite) {
+    this.x = x;
+    this.y = y;
+    this.sprite = sprite;
+};
+
+// Method for character update
+Character.prototype.update = function() {};
+
+// Draw the character on the screen, required method for game
+Character.prototype.render = function() {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+//----------------------------------------------
+// Constructor for create enemy
+const Enemy = function(x, y, sprite, speed, player) {
```

Comment:
Enemy constructor receives the `player` argument but it never used. Other class' methods still refer to the global variable `player`

Reply:
I fixed this thing